### PR TITLE
Load Github content as HTML.

### PR DESCRIPTION
**Pull request by nisevi**
Issue associated: https://github.com/AgileVentures/WebsiteOne/issues/2251

### CODE REVIEW

File: app/jobs/github_static_pages_job.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  end
+
+  def run
+    repo_content = client.contents('agileventures/agileventures')
+    md_pages = repo_content.select{|page| page if page[:path] =~ /\.md$/i}
+    md_pages.each do |page|
+      page_content = client.contents('agileventures/agileventures', path: page[:path])
+      markdown = Base64.decode64(page_content[:content])
+      # TODO: identify static page that is gonna get updated
+      static_page = StaticPage.find_by_slug('about-us')
```
Comment (nisevi) [inline app/jobs/github_static_pages_job.rb]: @tansaku I need to identify the pages that we are getting from [here](https://github.com/AgileVentures/AgileVentures) with the ones that we already have in our database :+1:

File: app/jobs/github_static_pages_job.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      unless static_page.nil?
+        static_page.body = MarkdownConverter.new(markdown).as_html
+        static_page.save!
+      end
+      # TODO: create a new static page if we didn't find it?
```
Comment (nisevi) [inline app/jobs/github_static_pages_job.rb]: @tansaku if a page doesn't exist we are creating a new one, right?

**Comment by nisevi** [comment]
@tansaku I'm getting this error when trying to convert some of the pages we have:

```ruby
bera fetch_github:content_for_static_pages
Trying to convert this page: ABOUT_US.md caused the issue!
"\xE2" from ASCII-8BIT to UTF-8
########################################################################################################
Trying to convert this page: CODE.md caused the issue!
"\xE2" from ASCII-8BIT to UTF-8
########################################################################################################
Trying to convert this page: CODE_OF_CONDUCT.md caused the issue!
"\xE2" from ASCII-8BIT to UTF-8
########################################################################################################
Trying to convert this page: GROW.md caused the issue!
"\xE2" from ASCII-8BIT to UTF-8
########################################################################################################
Trying to convert this page: JOINING_A_PROJECT.md caused the issue!
"\xE2" from ASCII-8BIT to UTF-8
########################################################################################################
Trying to convert this page: LEARN.md caused the issue!
"\xE2" from ASCII-8BIT to UTF-8
########################################################################################################
Trying to convert this page: PAIR.md caused the issue!
"\xC3" from ASCII-8BIT to UTF-8
########################################################################################################
Trying to convert this page: RUNNING_A_PROJECT.md caused the issue!
"\xC2" from ASCII-8BIT to UTF-8
########################################################################################################
Coverage report generated for RSpec to /home/nisevi/documents/agileventures/WebsiteOne/coverage. 197 / 3001 LOC (6.56%) covered.
```

I'm doing a little research to see how to approach this issue :+1:

**Comment by nisevi** [comment]
All errors regarding conversion were fixed :+1:

**Comment by nisevi** [comment]
@tansaku please let me know if you any suggestions about the approach I have taken :+1:

**Comment by tansaku** [comment]
this is looking great @nisevi - my only worry about merging it is that if we run into trouble we might block ourselves from releasing stuff like Nick's fix on the karma stuff.  If we can get that out on to staging first, then we can get this into develop without blocking ... we could also deploy this branch direct to develop - can maybe show you how to do that on Monday ...

Have you run the rake task for yourself locally on the production data?

**Comment by tansaku** [comment]
and coverage looks weird ...

**Comment by nisevi** [comment]
@tansaku yup I still have to test this rake task.

Yes I have run this task hitting the production data since is hitting the AV/AV repository and it is working :+1:

**Comment by tansaku** [comment]
@nisevi all done now?

**Comment by nisevi** [comment]
@tansaku no.

**Comment by tansaku** [comment]
@nisevi so you've got the css to where you're reasonably happy with it?

**Comment by nisevi** [comment]
@tansaku yes :+1:

**Comment by tansaku** [comment]
cool - I need to finish the deploy of the latest code on staging, and then maybe we can move this onto develop ...

**Pull request merged**